pvacseq/lib/pvacseq_utils.py

```diff
@@ -19,3 +23,50 @@ def iedb_to_prediction_method_lookup_dict():
 def iedb_prediction_methods():
     prediction_method_lookup_dict = iedb_to_prediction_method_lookup_dict()
     return sorted(prediction_method_lookup_dict.keys())
+
+def valid_allele_names_for_method(iedb_method):
+    #Ultimately we probably want this method to call out to IEDB but their command is currently broken
+    #curl --data "method=ann&species=human" http://tools-api.iedb.org/tools_api/mhci/
+    base_dir               = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
+    iedb_alleles_dir       = os.path.join(base_dir, 'iedb_alleles')
+    iedb_alleles_file_name = os.path.join(iedb_alleles_dir, "%s.tsv" % iedb_method)
+    alleles = set()
+    with open(iedb_alleles_file_name) as iedb_alleles_file:
```

Comment:
No, you're totally right. Reading it in from the file is pretty spiffy but once we change it to be requests, those take quite some time. 

I've been going back and forth on implementing this as requests since the list of alleles should be static for each IEDB version so we'd only need to worry about updating it when a new IEDB version comes out, at which point we'd probably have to make other changes anyway.

For now I will see if I can implement a solution where this gets stored in a variable after the first parsing.